console/src/lineplot/ontology.tsx

```diff
@@ -38,12 +37,10 @@ const TreeContextMenu: Ontology.TreeContextMenu = ({
     })();
   };
 
-  const clusterKey = Cluster.useSelectActiveKey();
-
   const handleRename = (): void => Tree.startRenaming(resources[0].key);
 
   const handleCopyLink = (): void => {
-    const toCopy = `synnax://cluster/${clusterKey}/lineplot/${resources[0].id.key}`;
+    const toCopy = `synnax://cluster/${client.key}/lineplot/${resources[0].id.key}`;
```

Comment:
Use `clusterKey` not `client.key`

Reply:
This won't work. The client key is not equal to the cluster key. Use the cluster key instead.